packages/vgrammar-projection/package.json

```diff
@@ -44,17 +44,18 @@
     "@visactor/vgrammar-util": "workspace:0.3.2",
     "@visactor/vgrammar": "workspace:0.3.3",
     "@visactor/vutils": "~0.11.1",
-    "d3-geo": "^1.11.6"
+    "d3-geo": "^3.0.1"
   },
   "devDependencies": {
+    "d3-array": "2.5.0 - 3",
```

Comment:
double check here, is d3-array a dev dependencies?

Reply:
yes, only for local test, jest will throw error if we don't import this